src/constants/constants.js

```diff
@@ -556,10 +556,12 @@ export const CONTRACTS = {
     STAKING_CONTRACT: 'StakingContract',
     PROFILE_CONTRACT: 'ProfileContract',
     HUB_CONTRACT: 'HubContract',
+    // TODO: Update with new commit Managers
     COMMIT_MANAGER_V1_U1_CONTRACT: 'CommitManagerV1U1Contract',
     SERVICE_AGREEMENT_V1_CONTRACT: 'ServiceAgreementV1Contract',
     PARAMETERS_STORAGE_CONTRACT: 'ParametersStorageContract',
     IDENTITY_STORAGE_CONTRACT: 'IdentityStorageContract',
+    Log2PLDSF: 'Log2PLDSF',
```

Comment:
This won't work as we don't listen to the events from this contract + it should be handled in a different way comparing to other contracts as we get parameters with one getter, but set them with different setters